scripts/dev/create_kustomize_dev_yaml.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import sys
+import yaml
+
+from dev_config import load_config
+
+
+def main() -> int:
+    dev_config = load_config()
+    dev_yaml = {
```

Comment:
kustomize does not provide a native mechanism to substitute values in patch files. This isn't an ideal solution, but I think it does exactly what we require for now.